.appveyor.yml

```diff
@@ -43,7 +43,7 @@ install:
 before_build:
   # setup env
   - CALL "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat"
-  - SET PATH=%PATH%;c:\qbt\qt5_64\bin;%CACHE_DIR%\jom;
+  - SET PATH=%PATH%;C:\Qt\5.15.2\msvc2019_64\bin;%CACHE_DIR%\jom
```

Comment:
@glassez 

_Non-blocking comment_

Since we're now using built-in Qt binaries in Appveyor, why not ditch the manually updated `https://builds.shiki.hu/appveyor/qbt_libraries_64.7z` completely? Appveyor also has recent enough Boost and OpenSSL versions.

You'd just need to build libtorrent as well as part of the CI run, but that's pretty easy and quick.

Reply:
>Since we're now using built-in Qt binaries in Appveyor, why not ditch the manually updated `https://builds.shiki.hu/appveyor/qbt_libraries_64.7z` completely? Appveyor also has recent enough Boost and OpenSSL versions.

We used our own binaries, apparently because AppVeyor did not have suitable ones (I can't say for sure, because I have never delved into these matters before). If AppVeyor can provide them to us now, why not use them? I don't mind, but I don't intend to contribute to it yet.
BTW, I didn't find `jom` here. Of course, we can still use `nmake`, although it is not so effective.

>You'd just need to build libtorrent as well as part of the CI run, but that's pretty easy and quick.

Whatever you said before, AppVeyor still wins in speed, because it compiles nothing but qBittorrent. Of course, if you add only libtorrent, it still shouldn't slow down much.
I wonder if we could use AppVeyor's cache so as not to rebuild the same version of libtorrent each time.

Reply:
> We used our own binaries, apparently because AppVeyor did not have suitable ones (I can't say for sure, because I have never delved into these matters before). If AppVeyor can provide them to us now, why not use them?

:+1:

> BTW, I didn't find `jom` here. Of course, we can still use `nmake`, although it is not so effective.

... or just use CMake + Ninja (or the native VS2019 built tools)

> Whatever you said before, AppVeyor still wins in speed, because it compiles nothing but qBittorrent. Of course, if you add only libtorrent, it still shouldn't slow down much.

GHA is still faster, all other things being equal (e.g. both using caches of the dependencies). I've noticed that lately Windows builds on GHA have not been caching dependencies properly though (which of course makes GHA slower on Windows/macOS), need to look into that (unfortunately, among other things, available cache sizes in CI services is still quite small).

Appveyor's slowness is perhaps more evident in other projects. libtorrent is a good example. If you've ever contributed there, you know just how much slower it is than GHA. It sometimes takes hours to even _start_ the jobs, I guess because the number of simultaneous jobs allowed is really small.

> I wonder if we could use AppVeyor's cache so as not to rebuild the same version of libtorrent each time.

Probably, this is what GHA does already (modulo the issue that has apparently surfaced recently).

Reply:
> I don't mind, but I don't intend to contribute to it yet.

No problem, I just wanted to raised awareness to this. I've always wanted qBittorrent to not use our own sparingly-updated, manually-built dependency bundle in CI.

Reply:
>... or just use CMake + Ninja (or the native VS2019 built tools)

`qmake` is still supported so we need to check it on CI (especially since Travis is gone so we have no more qmake-based builds).

Reply:
@glassez 

> `qmake` is still supported so we need to check it on CI (especially since Travis is gone so we have no more qmake-based builds).

Sure, it is probably more efficient to wait for the transition to Qt6 before making further changes to Appveyor, because then you'll be able to also drop the usage of `qmake` in Appveyor without issue.

Also, `jom.exe` may indeed be included as part of the official Qt distribution, under `Tools\QtCreator\bin`. It might not be available in Appveyor though, if they did not install that component of the distribution.